crypto/key_not_openssl.go

```diff
@@ -0,0 +1,40 @@
+// +build !openssl
+
+package crypto
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/rsa"
+
+	btcec "github.com/btcsuite/btcd/btcec"
+	"golang.org/x/crypto/ed25519"
+)
+
+// KeyPairFromKey generates a new private and public key from an input private key
+func KeyPairFromKey(priv crypto.PrivateKey) (PrivKey, PubKey, error) {
```

Comment:
How about we name this `KeyPairFromStdKey` to more accurately describe what’s going on?

Reply:
done